homematic/README.md

```diff
@@ -2,7 +2,7 @@
 
 HomeMatic central based on OCCU.
 
-![Supports aarch64 Architecture][aarch64-shield] ![Supports amd64 Architecture][amd64-shield] ![Supports armhf Architecture][armhf-shield] ![Supports armv7 Architecture][armv7-shield] ![Supports i386 Architecture][i386-shield]
```

Comment:
We should not remove them, instead, leave them and mark them `red` and `no` in the icons listed in the bottom off the file.